src/setfit/modeling.py

```diff
@@ -23,6 +25,43 @@
 MODEL_HEAD_NAME = "model_head.pkl"
 
 
+class SetFitDataset(Dataset):
+    def __init__(self, x, y, tokenizer, max_length=32):
```

Comment:
Small nit: could we add some types here? For things like `tokenizer`, we can import them with `typing.TYPE_CHECKING` and then wrap the type in quotes " ... "

Reply:
okay, I will add them in the next commit

Reply:
Just added it!